api_li3ds/apis/foreignpc.py

```diff
@@ -432,14 +432,44 @@ def post(self):
                 res = Database.query_asdict(req, {'schema_quat': schema_quat, 'srid': srid})
             pcid = res[0]['pcid']
 
+            # See following link for (roll, pitch, heading) to quaternion conversion
+            # https://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/19770024290.pdf
+            # rotation sequence (10) of the NASA document (appendix A) is used, with a slight
+            # change because of an axis inversion
+            # the quaternion we use is (0, 1/sqrt(2), 1/sqrt(2), 0) * nasa quaternion
+            # theta1 is roll
+            # theta2 is pitch
+            # theta3 is heading
```

Comment:
I think there's a mistake here:

- roll is phi (wikipedia) is theta3 (nasa)
- pitch is theta (wikipedia) is theta2 (nasa)
- heading is psi (wikipedia) is theta1 (nasa)

To be checked.

Reply:
I can confirm that there's a mistake here. I'll fix it.